nextflow.config

```diff
@@ -0,0 +1,87 @@
+manifest {
+    name = 'SNP&SEQ Run folder QC pipeline'
+    author = 'SNP&SEQ Bioinformatics Team; Mahesh Binzer-Panchal'
+    homePage = ''
+    description = 'A Nextflow run folder QC pipeline for SciLifeLab SNP&SEQ platform'
+    mainScript = 'main.nf'
+    nextflowVersion = '>=19.10.0'
+    version = '1.0'
+}
+
+profiles {
+
+    debug { process.beforeScript = 'env'}
+
+    dev {
+        executor.name = 'local'
+        executor.memory = '12 G'
+    }
+
+    irma {
+        process {
+            shell = ['/bin/bash', '-euo', 'pipefail']
+            errorStrategy = { task.exitStatus in [255] ? 'retry' : 'terminate' }
+            maxRetries = 2
+            scratch = true
+            executor = 'slurm'
+            clusterOptions = { "-A $params.project" }
+            cpus = 1
+            memory = '8G'
+            withName: 'fastq_screen' {
+                memory = '4 G'
+            }
+            withName: 'get_QC_thresholds' {
+                errorStrategy = 'ignore'
+            }
+        }
+    }
+
+    snpseq {
+        process {
+            shell = ['/bin/bash', '-euo', 'pipefail']
+            errorStrategy = { task.exitStatus in [255] ? 'retry' : 'terminate' }
+            maxRetries = 2
+            scratch = true
+            executor = 'local'
+            cpus = 1
+            memory = '32G'
+            withName: 'fastq_screen' {
+                memory = '4 G'
+            }
+            withName: 'get_QC_thresholds' {
+                errorStrategy = 'ignore'
+            }
+        }
+    }
+
+    singularity {
+        singularity {
+            enabled = true
+            autoMounts = true
+        }
+        process {
+            withName: 'fastqc' {
+                container = "docker://quay.io/biocontainers/fastqc:0.11.8--2"
+            }
+            withName: 'fastq_screen' {
+                container = "docker://quay.io/biocontainers/fastq-screen:0.13.0--pl526_1"
+            }
+            withName: 'get_QC_thresholds' {
+                container = "shub://Molmed/summary-report-development:checkqc"
+            }
+            withName: 'get_metadata' {
+                container = "shub://Molmed/summary-report-development:checkqc"
+            }
+            withName: 'interop_summary' {
+                container = "docker://quay.io/biocontainers/illumina-interop:1.0.25--0"
+            }
+            withName: 'multiqc_per_flowcell' {
+                container = "shub://Molmed/summary-report-development:multiqc"
+            }
+            withName: 'multiqc_per_project' {
+                container = "shub://Molmed/summary-report-development:multiqc"
```

Comment:
Same here: `docker://ewels/multiqc:v1.7`

Reply:
changed to use biocontainers image